src/ConfigUpdater.php

```diff
@@ -0,0 +1,279 @@
+<?php
+
+namespace Drupal\commerce;
```

Comment:
We are adding ConfigUpdater, ConfigUpdaterInterface, plus the result object to src/.
Since we have more than one file connected by a common purpose, it makes sense to make a namespace for them.
Drupal\commerce\Config perhaps?